frontend/pages/Store/index.tsx

```diff
@@ -0,0 +1,2 @@
+export { default } from '@templates/page';
+export { getServerSideProps } from '@templates/page/server';
```

Comment:
This file is not necessary.. We can make the catch all segment [optional](https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes#optional-catch-all-segments) by renaming `frontend/pages/Store/[...slug].tsx` to `frontend/pages/Store/[[...slug]].tsx`

Reply:
TIL! But I'd still rather have this two line file than have someone need to know about the particulars of what double-brackets means in a file name. Actually, for that reason, I think I'll rename this to just `pages/Store.tsx` so it's even more obvious.

Reply:
I forgot about it too but we shouldn't get in the habit of creating extra files just in case someone doesn't know a feature of the framework we're using. I'm OK with putting it at just /Store for now since we're not using other pages, but we'll probably want to revisit this eventually when we do want to put up more pages.